src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsScaleUnitNode.ps1

```diff
@@ -0,0 +1,144 @@
+<#
```

Comment:
@deathly809 do we have/need any test coverage surrounding this new cmdlet?

Reply:
We have tested this internally and with OEM partners.  I don't have a $25K server to test on :(